CONTRIBUTING.md

```diff
@@ -37,6 +37,9 @@ sh scripts/add-chain.sh frontier
 ### 3. Understand output
 The tool will write the following data:
 - The main configuration source, with genesis data, and address of onchain system configuration.
+> **Note**
+> Hardfork override times will be included. For standard chains, you must have all hardforks activated (see the neighbouring superchain.yaml file). It is not possible to override a superchain-wide hardfork time with `nil`.
```

Comment:
So we no longer use the superchain.yaml file as a default and now always "duplicate" the timestamps in the local file?

Reply:
We do use the `superchain.yaml` file as a default, but override it when there are non-nil values in the `chain-name.yaml` file. It is not necessary to duplicate the values in general.

We should only include the timestamp locally when it differs, yes.

Reply:
> We should only include the timestamp locally when it differs, yes.

This PR always includes it though IIUC?

Reply:
Yes, correct. I'll file an issue to followup to change that.

Reply:
#166